crates/polars-core/src/series/arithmetic/list_borrowed.rs

```diff
@@ -53,23 +53,112 @@ fn lists_same_shapes(left: &ArrayRef, right: &ArrayRef) -> bool {
     }
 }
 
+/// Arithmetic operations that can be applied to a Series
```

Comment:
Is there some existing code that does this? There's a whole bunch of _similar_ things but I may've missed an actual implementation.